ansible/stage2.yml

```diff
@@ -1,13 +1,32 @@
 ---
-- name: Execute getbestchainlock
+
+#We need to check first to ensure mixing is not enabled from a previous run, if it tries to do it again it will fail (thus not idempotent)
```

Comment:
Where is this check?

Reply:
Ah this got changed, turns out if we put coinjoinmultisession in the dashd conf it starts automatically and works idempotently since createwallet will simply come back as OK if it's already there, I will remove this comment